### PR TITLE
A pass over encoding.md

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

File: site/docs/encoding.md [hunk 1]
```diff
@@ -61,64 +88,88 @@ The following table lists supported channels for each mark type.
...
-After the specified top-level `transform`s are executed, inline transforms are executed in this order: binning, time unit conversion, aggregation, and sorting.
+<!--TODO: revise description in this table-->
 
 | Property      | Type          | Description    |
 | :------------ |:-------------:| :------------- |
-| [bin](bin.html) | Boolean &#124; Object        | Boolean flag / configuration object for binning.   |
-| [timeUnit](timeunit.html)| String        | Property for converting time unit.            |
-| [aggregate](aggregate.html) | String        | Aggregation function for the field (e.g., `mean`, `sum`, `median`, `min`, `max`, `count`).  |
-| [sort](sort.html) | String &#124; Object        | Sort order for a particular field.  This can be string (`"ascending"`, `"descending"`, or `"unsorted"`) or a sort field definition object for sorting by an aggregate calculation of a specified sort field.  If unspecified, the default value is `ascending`. |
+| [bin](bin.html)<sup>1</sup> | Boolean &#124; Object | Boolean flag / configuration object for binning.    |
```
Comment (domoritz) [inline site/docs/encoding.md]: Kramdown has native support for footnotes. http://kramdown.gettalong.org/syntax.html#footnotes

Comment (kanitw) [inline site/docs/encoding.md]: I know but that will put footnote at the end of the page, which is obviously too far away in this case.

**Comment by kanitw** [comment]
Just did another pass.  I think once we fill in the placeholder with examples this page should be good to go.  (Feel free to merge this, we can add the examples later.)

File: site/docs/encoding.md (hunk 1 above)
Comment (domoritz) [inline site/docs/encoding.md]: Meh. Okay.

**Pull request merged**